numpy/distutils/system_info.py

```diff
@@ -1650,8 +1685,8 @@ def get_atlas_version(**config):
 
 class lapack_opt_info(system_info):
     notfounderror = LapackNotFoundError
-    # List of all known BLAS libraries, in the default order
-    lapack_order = ['mkl', 'openblas', 'flame', 'atlas', 'accelerate', 'lapack']
+    # List of all known LAPACK libraries, in the default order
+    lapack_order = ['mkl', 'openblas', 'flame', 'atlas', 'lapack']
```

Comment:
If you take accelerate out here, you need to
- remove it from one of the travis configurations
- disable the test on azure that checks for an error when using accelerate